src/Neo/SmartContract/Native/NativeContract.cs

```diff
@@ -288,13 +283,26 @@ internal bool IsInitializeBlock(ProtocolSettings settings, uint index, out Hardf
 
                 if (activeIn == index)
                 {
-                    hardfork = hf;
-                    return true;
+                    hfs.Add(hf);
                 }
             }
 
+            // Return all initialize hardforks
+            if (hfs.Count > 0)
+            {
+                hardforks = hfs.ToArray();
+                return true;
+            }
+
+            // If is not configured, the Genesis is the a initialized block
```

Comment:
While we're here, the comment may be fixed:
```suggestion
            // If is not configured, the Genesis is an initialization block.
```